Source/WebCore/accessibility/AccessibilityObject.h

```diff
@@ -134,7 +133,8 @@ class AccessibilityObject : public AXCoreObject {
 #endif
     bool isMediaTimeline() const { return false; }
     virtual bool isSliderThumb() const { return false; }
-    bool isLabel() const { return isAccessibilityLabelInstance() || labelForObjects().size(); }
+    virtual bool isLabel() const { return false; }
+    bool isLabelForObject() const;
```

Comment:
Any reason this isn't just inlined?

`bool isLabelForObject() const { return labelForObjects().size(); }`